src/components/Chart/renderAxis.tsx

```diff
@@ -139,8 +144,7 @@ function getAxisProps(
     }
 }
 
-// TODO: Talvez o problema da outra issue esteja aquitalvez o problema esteja aqui
-const convertTickProps = (props: any): TickProps => ({
+export const convertTickProps = (props: any): TickProps => ({
```

Comment:
Since it is now being used by more than one file, it could be moved to a common file, like `util.ts` or `model.ts`

Reply:
:point_up:

Reply:
Seems good. Just remember to follow the bold code patterns, not the PEC ones.